### PR TITLE
[C] Migrate RG styles

### PR DESCRIPTION
**Pull request by 1aurend**
Migrates styles from `theme/components/frontend/reading-group` and `theme/components/frontend/collecting` to styled.

### CODE REVIEW

**Review by dananjohnson — CHANGES_REQUESTED** [review]
I gave the code a quick look and it seems fine, but I did notice a couple visual regressions when I started testing this. Make sure you go through every route and breakpoint and compare to what you see on Edge. Once you've done that and addressed any issues you see, go ahead and merge it.

Local:
Screen Shot 2022-04-07 at 10 14 00 AM
Edge:
Screen Shot 2022-04-07 at 10 14 07 AM


Local:
Screen Shot 2022-04-07 at 10 15 24 AM
Edge:
Screen Shot 2022-04-07 at 10 15 30 AM

**Comment by 1aurend** [comment]
@dananjohnson I went through the routes, compared, and added a few fixes. The PR needs your approval before I can merge. Thx!

**Review by dananjohnson — APPROVED** [review]

**Pull request merged**
